projects/swimlane/ngx-ui/src/lib/components/button/button.component.ts

```diff
@@ -1,7 +1,7 @@
 import { Component, Input, ViewEncapsulation, OnInit, OnChanges, HostListener, ChangeDetectionStrategy } from '@angular/core';
+import { coerceBooleanProperty } from '@angular/cdk/coercion';
```

Comment:
how nice of angular to include this :)

Reply:
hopefully this fixes that weird issue you were seeing when integrating with platform...